src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -703,4 +733,105 @@ private void workAroundHttpClientCloneBug() {
             alignment.setCigar(new ArrayList<>(alignment.getCigar()));
         }
     }
+
+    /**
+     * Return a SAM string representation of this read.
+     */
+    private String createSAMString() {
+        final String SAM_FIELD_SEPARATOR = "\t";
+        final String SAM_ATTRIBUTE_SEPARATOR = ":";
+
+        final String contigName = getContig();
+        final StringBuilder sb = new StringBuilder();
+
+        sb.append(getName());
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(Integer.toString(ReadUtils.getSAMFlagsForRead(this)));
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(null == contigName ? SAMRecord.NO_ALIGNMENT_REFERENCE_NAME : contigName);
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(Integer.toString(getStart()));
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(Integer.toString(getMappingQuality()));
+        sb.append(SAM_FIELD_SEPARATOR);
+        sb.append(getCigar().toString());
+        sb.append(SAM_FIELD_SEPARATOR);
+
+        if (!isPaired()) {
+            sb.append("=");
```

Comment:
Shouldn't this be set to `*` if the read is unpaired?  That's my reading of the sam spec.

```
7. RNEXT: Reference sequence name of the primary alignment of the NEXT read in the template. For
the last read, the next read is the first read in the template. If @SQ header lines are present, RNEXT (if
not ‘*’ or ‘=’) must be present in one of the SQ-SN tag. This field is set as ‘*’ when the information is
unavailable, and set as ‘=’ if RNEXT is identical RNAME. If not ‘=’ and the next read in the template
has one primary mapping (see also bit 0x100 in FLAG), this field is identical to RNAME at the primary
line of the next read. If RNEXT is ‘*’, no assumptions can be made on PNEXT and bit 0x20.
```